src/sentry/incidents/endpoints/serializers.py

```diff
@@ -325,6 +326,15 @@ class Meta:
             "resolve_threshold": {"required": False},
         }
 
+    def validate_query(self, query):
+        query_terms = query.split()
+        for query_term in query_terms:
+            if query_term in unsupported_queries:
+                raise serializers.ValidationError(
+                    "Unsupported Query: We do not currently support the {} query".format(query_term)
+                )
+        return query
+
```

Comment:
I think this would fail if you had a query like `some stuff "some enclosed stuff release:latest"`. That's probably not all that important or likely to happen, but I think ideally what we would do here is have some way to pass search terms we want to block to the `event_search` parser and have it raise an error. 

For now, I'm ok with what we're doing here, but maybe it's worth leaving a comment near `unsupported_queries` so that we don't just keep adding to this.